.circleci/config.yml

```diff
@@ -9,24 +9,24 @@ workflows:
     jobs:
       - ios/test:
           name: Test
-          xcode-version: "10.2.0"
+          xcode-version: "11.0"
           carthage-update: true
           carthage-working-directory: Example
           bundle-install: false
           pod-install: false
           workspace: Aztec.xcworkspace
           scheme: AztecExample
           device: iPhone XS
```

Comment:
This should be `iPhone 11`. Xcode 11 does not come with an iPhone XS simulator by default.